test/cxx/Core/ApplicationPool/ProcessTest.cpp

```diff
@@ -118,8 +121,10 @@ namespace tut {
 
 			args["spawner_creation_time"] = 0;
 
-			ProcessPtr process(context.processObjectPool.construct(
-				&groupInfo, result, args), false);
+			Process *p = context.processObjectPool.malloc();
```

Comment:
Why did you change this to use `pool.malloc()`?

Reply:
Because the change in the number of arguments makes using the old way no longer work.